### ISSUE TITLE
[ic-search-bar / ic-select] aria-expanded="false" is not an allowed attribute

### ISSUE THREAD

**Issue by GCHQ-Developer-530** [mi6/ic-ui-kit#1912]
## Summary of the bug 
Title: WCAG 4.1.2: Ensures ARIA attributes are allowed for an element's role (ic-search-bar,ic-text-field,#ic-search-bar-input-1)
Tags: Accessibility, WCAG 4.1.2, aria-allowed-attr

Issue: Ensures ARIA attributes are allowed for an element's role (aria-allowed-attr - https://accessibilityinsights.io/info-examples/web/aria-allowed-attr)

Snippet: <input id="ic-search-bar-input-1" name="ic-search-bar-input-1" type="text" class="" placeholder="Search" aria-label="" aria-describedby="ic-search-bar-input-1-assistive-hint" aria-invalid="false" aria-expanded="false" autocomplete="off" autocapitalize="off" spellcheck="false" inputmode="search" aria-haspopup="listbox">

How to fix: 
Fix all of the following:
  ARIA attribute is not allowed: aria-expanded="false"

====

## 🪜 How to reproduce 
Tell us the steps to reproduce the problem: 
1. Go to any search bar / select example that has options
3. Run accessibility fast pass
4. See the error 

## 🧐 Expected behaviour 
There should be no accessibility violations.

## Additional info 
I tried setting the aria-expanded value to undefined instead of "false" and this fixes this issue but brings up many more accessibility failures.
